pyemu/utils/pst_from.py

```diff
@@ -1873,8 +2006,12 @@ def add_parameters(
                             "using ExpVario with contribution=1 "
                             "and a=(pp_space*max(delr,delc))"
                         )
-                        # set up a default
-                        pp_dist = pp_space * float(
+                        # set up a default - could probably do something better if pp locs are passed
```

Comment:
I guess all fairly arbitrary, but could just use max separation of adjacent pp if pp_locs passed? But is that really any more "correct".

Reply:
True - I'll add that bit